File: Server/App_Code/DatabaseConnect.vb
Imports System.Data.SQLite
Public Class DatabaseConnect
    Dim Database As SQLiteConnection
    Dim Command As New SQLiteCommand
    Public Sub DatabaseOpen()
        Database = New SQLiteConnection("Data Source=" + Config.GetAppPath() + "\Server\Database.db; Version=3;")
        Database.Open()
    End Sub

    Public Sub UpdateViewValue(TableName As String, ItemID As String, Count As String)
        Command = Database.CreateCommand()
        Command.CommandText = "UPDATE " + TableName + " SET Viewed='" + Count + "' WHERE ID=" + ItemID
        Command.ExecuteNonQuery()
    End Sub

    Public Function GetCountItems(NameTable As String) As Integer
        DatabaseOpen()
        Command = Database.CreateCommand()
        Command.CommandText = "SELECT Count (*) From " + NameTable
        Dim ReadItem = Command.ExecuteReader()
        ReadItem.Read()
        Dim Item = ReadItem(0)
        ReadItem.Close()
        Database.Dispose()
        Return Item
    End Function
    Public Function GetItemID(NameTable As String, ItemName As String) As String
        Command = Database.CreateCommand()
        Command.CommandText = "SELECT * FROM " + Config.CategoryTable + " WHERE Name LIKE '" + ItemName + "'"
        Dim ReadItem = Command.ExecuteReader()
        While ReadItem.Read()
            Dim Item = ReadItem.Item("ID").ToString
            ReadItem.Close()
            Return Item
        End While
        Return ""
    End Function
    Public Function GetItemByID(NameTable As String, ItemID As String, ItemProperty As String) As String
        DatabaseOpen()
        Command = Database.CreateCommand()
        Command.CommandText = "SELECT * FROM " + NameTable + " WHERE ID=" + ItemID
        Dim ReadItem = Command.ExecuteReader()
        While ReadItem.Read()
            Dim Item = ReadItem.Item(ItemProperty).ToString
            ReadItem.Close()
            Database.Dispose()
            Return Item
        End While
        Return ""
    End Function

    Public Sub DatabaseClose()
        Database.Dispose()
    End Sub
End Class

File: Server-CS/App_Code/DatabaseConnect.cs
using System.Diagnostics;
using System.Data.SQLite;

public class DatabaseConnect
{
    SQLiteConnection Database = new SQLiteConnection();
    SQLiteCommand Command = new SQLiteCommand();
    public DatabaseConnect()
    {

    }
    void DatabaseOpen()
    {
        string DBPath = Utilites.GetAppPath() + @"\" + "Server-CS" + @"\" + "Database.db";        
        Database = new SQLiteConnection("Data Source=" + DBPath + "; Version=3;");
        Database.Open();
    }
    public int GetCountItems(string NameTable)
    {
        DatabaseOpen();
        Command = Database.CreateCommand();
        Command.CommandText = "SELECT Count (*) From " + NameTable;
        SQLiteDataReader ReadItem = Command.ExecuteReader();
        ReadItem.Read();
        int Item = 0;
        int.TryParse(ReadItem[0].ToString(), out Item);
        ReadItem.Close();
        Database.Dispose();
        return Item;
    }
    public string GetItemByID(string NameTable, int ItemID, string Prop)
    {
        DatabaseOpen();
        Command = Database.CreateCommand();
        Command.CommandText = "SELECT * FROM " + NameTable + " WHERE ID=" + ItemID.ToString();
        SQLiteDataReader ReadItem = Command.ExecuteReader();
        while(ReadItem.Read())
        {
            int NumberColumn = ReadItem.GetOrdinal(Prop);
            string Item = ReadItem.GetValue(NumberColumn).ToString();
            ReadItem.Close();
            Database.Dispose();
            return Item;
        }
        return "";
    }
}